src/TicketMaster/Discovery/Event.php

```diff
@@ -17,19 +19,19 @@ class Event extends Api\AbstractApi
     public function findAll(): array
     {
         $response = $this->getClient()
-            ->get('discovery/v2/events?apikey='.$this->getKey().'&size=2');
+            ->get('discovery/v2/events?apikey='. $this->getKey().'&size=2');
 
         $eventsJson = $response->getBody()->getContents();
-        $eventsResponseArray = \GuzzleHttp\json_decode($eventsJson, true);
+        $eventsResponseArray = json_decode($eventsJson, TRUE);
```

Comment:
The PHP constants true, false, and null MUST be in lower case. http://www.php-fig.org/psr/psr-2/#2-5-keywords-and-true-false-null